### DISCUSSION TITLE
Set Error in object

### DISCUSSION THREAD

**Discussion by leandro-almeida98** [react-hook-form/react-hook-form#8174] [Q&A] [answered]
Hi, I have the following problem,
I'm trying to use the following code, but nothing happens,

I can't show the error of a field with name.object

CODESANDBOX of the problem below

https://codesandbox.io/s/react-hook-form-v7-setvalue-forked-f4jo8h?file=/src/index.jsx:0-1349

how to show the error of a field with the registered name of an object.field?

**Comment by Moshyfawn** [comment]
Hey, @leandro-almeida98!

It depends on your setup. Care to provide an isolated codesanbox?

**Reply by leandro-almeida98** [reply]
> Hey, @leandro-almeida98!
> 
> Depende da sua configuração. Importa-se de fornecer um codeanbox isolado?

Isolated code link

https://codesandbox.io/s/react-hook-form-v7-setvalue-forked-f4jo8h?file=/src/index.jsx:0-1349

**Reply by Moshyfawn** [reply]
What RHF version are you experiencing the issue in? Your repro is set to a beta version of V7.
```json
"dependencies": {
  "react-hook-form": "7.0.0-beta.13"
},
```

If that's the case, try upgrading to a stable version instead.

**Reply by leandro-almeida98** [reply]
> Em qual versão do RHF você está enfrentando o problema? Sua reprodução está configurada para uma versão beta da V7.
> 
> ```json
> "dependencies" : {
>    "react-hook-form" : " 7.0.0-beta.13 " 
> },
> ```
> 
> Se for esse o caso, tente atualizar para uma versão estável.

I updated to a stable version react-hook-form used, but I would like to know how I could show the error of this field registed how"profile.FirstName"

**Comment by Moshyfawn** [comment, marked as answer]
First off, to add an error message to your `required` field validation, you can just pass the error message to the `required` property.

Now, to get the nested object error, you want to access it with the object path you've defined with the `register` field name.

```diff
- {errors.firstName && <p>"Required Field"</p>}
+ {errors.profile?.firstName && <p>{errors.profile.firstName.message}</p>}
```

[Here's a working example](https://codesandbox.io/s/react-hook-form-v7-setvalue-forked-1m33rq?file=%2Fsrc%2Findex.jsx%3A520-592)

**Reply by leandro-almeida98** [reply]
Worked, thank you!
